admin-ui/src/components/token-dialog.tsx

```
@@ -77,10 +81,11 @@ export function TokenDialog(): React.JSX.Element {
 
       <DialogContent>
         <DialogHeader>
-          <DialogTitle>Admin token</DialogTitle>
+          <DialogTitle>{t("tokenDialog.title")}</DialogTitle>
           <DialogDescription>
-            Used as <code>x-admin-token</code> when calling <code>/api/admin/*</code>. Stored
-            in <code>sessionStorage</code>.
+            <Trans i18nKey="tokenDialog.description">
+              Used as <code>x-admin-token</code> when calling <code>/api/admin/*</code>. Stored in <code>sessionStorage</code>.
```

ESLint

[error] Insert `{"·"}⏎·············`

(prettier/prettier)